images/prometheus-redis-exporter/image.yaml

```diff
@@ -0,0 +1,6 @@
+status: experimental
```

Comment:
We should try to add tests to this so we can remove the `experimental` label. Is there anything you can think of that would test that this image works as expected?